File: JavaScript/abc016A.js
function Main(input) {
    const a = input.split(' ');
    const n = parseInt(a[0], 10);
    const m = parseInt(a[1], 10);
    const s = n % m;
    if(s == 0){
      console.log("YES");
    }else{
      console.log("NO");
    }
  }
  
  Main(require("fs").readFileSync("/dev/stdin", "utf8"));
  

File: Perl/abc016A.pl
#!/usr/bin/perl
 
my $list = <STDIN>;
my @n = split(/ /, $list);
chomp($n[0]);
chomp($n[1]);
my $m = $n[0] % $n[1];
if ($m == 0){
  print "YES\n";
}else{
  print "NO\n";
}
